### PR TITLE
Fixing issue #126 - Updating license

### PR DESCRIPTION
**Pull request by Anselmoo**
Fixing issue #126

* Update the license from `BSD` to `GNU`
* Add the classifier
* Be consistent in string notation; `'` -> `"`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,59 +22,65 @@
...
+        ]
+    },
     install_requires=requirements,
-    license="BSD (3-clause)",
+    license="GNU General Public License (GPL)",
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
-        'Natural Language :: English',
-        'Programming Language :: Python :: 3',
+        "License :: OSI Approved :: GNU General Public License (GPL)",
```
Comment (mrakitin) [inline setup.py:80]: Looks like the version of the license is v3. Should it be mentioned?

The following resources can be helpful:
- https://pypi.org/classifiers/
- https://spdx.org/licenses/

Also, is it 3-only or 3-or-later version? I was working on packaging `edrixs` for conda-forge (https://github.com/conda-forge/staged-recipes/pull/15729) with my SULI intern (still work-in-progress), so we may need to update the licensing information there, and conda-forge checks the license names to be compatible with the SPDX database, so it'd be good to understand whether we should use `GPL-3.0-only` or ` GPL-3.0-or-later`. Thanks.

Comment (Anselmoo) [reply setup.py:80]: @mrakitin thx for your quick response

Yes, you can do it! It was not clear for me based on your [license file](https://github.com/NSLS-II/edrixs/blob/master/LICENSE) between GPL and v1 -v3 are some major differences in the copyleft.

Comment (mrakitin) [reply setup.py:80]: https://github.com/NSLS-II/edrixs/blob/master/COPYING is the text of the license, so it's v3, but not sure if "only" or "3-or-later".

Comment (Anselmoo) [reply setup.py:80]: But I guess with `License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+) ` you save??

Comment (mrakitin) [reply setup.py:80]: I think so.

File: setup.py [hunk 2]
```diff
@@ -21,59 +22,65 @@
...
-        'Natural Language :: English',
-        'Programming Language :: Python :: 3',
+        "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+        "Development Status :: 2 - Pre-Alpha",
+        "Natural Language :: English",
+        "Programming Language :: Python :: 3",
     ],
-    ext_modules=[ext_fortran]
+    ext_modules=[ext_fortran],
 )
```
Comment (mrakitin) [inline setup.py:86]: Was this file formatted with `black`?

**Review by mrakitin — APPROVED** [review]
Looks good to me.

File: setup.py (hunk 2 above)
Comment (mrakitin) [reply setup.py:86]: No, we use `black` a lot for various projects, I personally like it.

**Comment by mpmdean** [comment]
Thanks!

**Pull request merged**